### ISSUE TITLE
Manual FFMPEG error

### ISSUE THREAD

**Issue by ghost** [mdhiggins/sickbeard_mp4_automator#138]
Hi,

First, this will make my daily work a huge breeze. I owe you beers on top of beers when I finally find you, or whatever your poison will be.

However, there seems to be a codec error, and I've been through it up and down and can't figure it out. I'm running this on Debian 7.5 (both 32 & 64 bit architectures) have compiled FFMPEG twice on each, can run it command line with codec flags included but when I run manual.py, I get this error. 

An unexpected error occurred, processing of this file has failed
Unknown ffmpeg error Encoder (codec none) not found for output stream #0:0

I have tried different variations of codecs i.e. x264 & libx264 as well as aac & libaac, libfaac, libfdk-aac but still, nothing.

Basically, it doesn't work. I've uninstalled FFMPEG so that I can install it with your script and that too doesn't make a difference. What could be the hitch?

Thanks.
